resources/plugin.rb

```diff
@@ -19,29 +19,28 @@
 # limitations under the License.
 
 property :grafana_cli_bin, String, default: '/usr/sbin/grafana-cli'
+property :plugin_url, String
```

Comment:
Is this expected to be in a certain format, e.g. http:// or just foo.com/plugins, can we add validation at all?

Reply:
It would appear that any valid url to a zip file is acceptable: https://grafana.com/docs/grafana/latest/administration/cli/#override-default-plugin-zip-url

i am open to suggestions on validations to apply.

Reply:
Happy with this for now :)